### DISCUSSION TITLE
EIP-3085 & EIP-3326

### DISCUSSION THREAD

**Discussion by leon-do** [Web3Auth/Web3Auth#429] [Feature Requests]
Will torus wallet entertain implementing EIP-3085 and EIP-3326?

In short they will be RPC calls `wallet_addEthereumChain` and `wallet_switchEthereumChain`

Similar to: https://github.com/orgs/Web3Auth/discussions/416

https://eips.ethereum.org/EIPS/eip-3085
https://eips.ethereum.org/EIPS/eip-3326

**Comment by itsinz** [comment]
@chaitanyapotti

**Comment by chaitanyapotti** [comment]
It's already supported
